fluid/PaddleCV/faster_rcnn/README.md

```diff
@@ -90,7 +90,7 @@ To train the model, [cocoapi](https://github.com/cocodataset/cocoapi) is needed.
 
 *  Use momentum optimizer with momentum=0.9.
 *  Weight decay is 0.0001.
-*  In first 500 iteration, the learning rate increases linearly from 0.00333 to 0.01. Then lr is decayed at 120000, 160000 iteration with multiplier 0.1, 0.01. The maximum iteration is 180000.
+*  In first 500 iteration, the learning rate increases linearly from 0.00333 to 0.01. Then lr is decayed at 120000, 160000 iteration with multiplier 0.1, 0.01. The maximum iteration is 180000. Also, we released a 2x model which has 360000 iterations and lr is decayed at 240000, 320000
```

Comment:
These hype-parameter should also add in the code.

Reply:
I add some comments in config.py